server/schedulers/hot_region.go

```diff
@@ -1212,3 +1251,24 @@ func getRegionStatKind(rwTy rwType, dim int) statistics.RegionStatKind {
 	}
 	return 0
 }
+
+func (bs *balanceSolver) preferPriority() [2]string {
+	priorities := bs.sche.conf.WritePriorities
+	if bs.rwTy == read {
+		priorities = bs.sche.conf.ReadPriorities
+	}
+	if len(priorities) == 1 {
```

Comment:
when will it happen? do we need to consider that len is zero?

Reply:
updated.